test/services/CategorisationServiceSpec.scala

```diff
@@ -631,142 +607,229 @@ class CategorisationServiceSpec extends SpecBase with BeforeAndAfterEach {
+            .set(AssessmentPage(testRecordId, 2), AssessmentAnswer.NoExemption)
+            .success
+            .value
+
+          categorisationService.calculateResult(
+            categorisationInfo,
+            userAnswers,
+            testRecordId
+          ) mustEqual Category1Scenario
+        }
       }
 
-      "if NIPHL is authorised and has NIPHL assessments when category 1 is answered no" in {
-        val assessment1 = CategoryAssessment(
-          "ass1",
-          1,
-          Seq(
-            Certificate("cert1", "cert1code", "cert1desc")
+      ".NIRMS" - {
+
+        "is not authorised and has NIRMS assessment, category 1 assessment and category 2 assessment when all are answered yes" in {
+          val nirmsAssessment = CategoryAssessment(
+            "nirms1",
+            2,
+            Seq(
+              Certificate(NirmsCode, "cert1code", "cert1desc")
+            )
           )
-        )
 
-        val assessment2 = CategoryAssessment(
-          "ass2",
-          1,
-          Seq(
-            Certificate(NiphlsCode, "cert2code", "cert2desc")
+          val assessment2 = CategoryAssessment(
+            "ass2",
+            1,
+            Seq(
+              Certificate("cert2", "cert2code", "cert2desc")
+            )
           )
-        )
 
-        val assessment3 = CategoryAssessment(
-          "ass1",
-          1,
-          Seq(
-            Certificate("cert3", "cert3code", "cert3desc")
+          val assessment3 = CategoryAssessment(
+            "ass3",
+            2,
+            Seq(
+              Certificate("cert3", "cert3code", "cert3desc")
+            )
           )
-        )
 
-        val categorisationInfo = CategorisationInfo(
-          "1234567890",
-          Seq(
-            assessment1,
-            assessment2,
-            assessment3
-          ),
-          Seq(assessment1, assessment2, assessment3),
-          None,
-          1,
-          isTraderNiphlsAuthorised = true
-        )
+          val categorisationInfo = CategorisationInfo(
+            "1234567890",
+            Seq(
+              nirmsAssessment,
+              assessment2,
+              assessment3
+            ),
+            Seq(assessment2, assessment3),
+            None,
+            1
+          )
+
+          val userAnswers = emptyUserAnswers
+            .set(CategorisationDetailsQuery(testRecordId), categorisationInfo)
+            .success
+            .value
+            .set(AssessmentPage(testRecordId, 0), AssessmentAnswer.Exemption)
+            .success
+            .value
+            .set(AssessmentPage(testRecordId, 1), AssessmentAnswer.Exemption)
+            .success
+            .value
+
+          categorisationService.calculateResult(
+            categorisationInfo,
+            userAnswers,
+            testRecordId
+          ) mustEqual Category2Scenario
+        }
+      }
+
+    }
+
+    "return StandardNoAssessments" - {
+
+      "if all answers are Yes" in {
```

Comment:
that does not return StandardNoAssessements.

Reply:
Not sure about this one

Reply:
Just need to sort out the titles.

need like 

"return Standard" - {
   "if all answers are Yes" in {
   
   }
   }
   
   "return StandardNoAssessments" - {
       "whatever test that is" in {
       
       }
      }

Reply:
I see. Got it :S

Reply:
Updated